pkg/process/checks/process.go

```diff
@@ -61,6 +61,9 @@ type ProcessCheck struct {
 
 	maxBatchSize  int
 	maxBatchBytes int
+
+	checkCount int64
+	skipAmount int64
```

Comment:
do we need int64? thoughts just using `uint32`?

Reply:
uint32 should work, only had int64 as I was using timestamps before